hv-rhel7.x/hv/tools/hv_vss_daemon.c

```diff
@@ -93,8 +95,11 @@ static int vss_operate(int operation)
 	while ((ent = getmntent(mounts))) {
 		if (strncmp(ent->mnt_fsname, match_dev, strlen(match_dev)) != 0)
 			continue;
-		if (strncmp(ent->mnt_fsname, match_loop, strlen(match_loop)) == 0)
+		if (stat(ent->mnt_fsname, &sb) == -1)
 			continue;
+		if (S_ISBLK(sb.st_mode))
```

Comment:
Any reason we can't && these two if statements into one?

if (S_ISBLK(sb.st_mode) && major(sb.st_rdev) == LOOP_MAJOR)